packages/lms/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@floating-ui/dom": "^1.2.8",
     "@mollify/cli": "^0.2.11",
+    "@mollify/lms": "^0.2.11",
```

Comment:
A project shouldn't install itself

Reply:
This is not something I have added myself, After running all the installations according to the Readme. This is not a change I have done by my self.

Reply:
I figured out what happened, you need to move the material install here and remove it from the root level package file - and remove this line.